CMakeLists.txt

```diff
@@ -15,12 +29,11 @@ else()
     set(PICO_SDK_PATH lib/pico-sdk)
 endif()
 
-if(DEFINED ENV{FREERTOS_KERNEL_PATH})
-    set(FREERTOS_KERNEL_PATH $ENV{FREERTOS_KERNEL_PATH})
+if(DEFINED ENV{FREERTOS_PATH})
+    set(FREERTOS_PATH $ENV{FREERTOS_PATH})
 else()
-    set(FREERTOS_KERNEL_PATH lib/freertos)
+    set(FREERTOS_PATH lib/freertos)
```

Comment:
we changed this from set(FREERTOS_KERNEL_PATH lib/freertos) to 
set(FREERTOS_KERNEL_PATH ${CMAKE_SOURCE_DIR}/lib/freertos)

that fixed the issue of it accidentally looking into build/lib/freertos instead of just lib/freertos